ENIGMAsystem/SHELL/Platforms/General/POSIX/Utility.cpp

```diff
@@ -129,10 +128,7 @@ void execute_program(std::string operation, std::string fname, std::string args,
 void execute_program(std::string fname, std::string args, bool wait) { execute_program("", fname, args, wait); }
 
 void url_open(std::string url, std::string target, std::string options) {
-  if (!fork()) {
-    execlp("xdg-open", "xdg-open", url.c_str(), NULL);
-    exit(0);
-  }
+  execute_program("xdg-open", url, false);
```

Comment:
Good change; writing a better `execlp`/`execvp` has been on my to-do list for ages.